src/olympia/promoted/tasks.py

```diff
@@ -49,29 +50,33 @@ def add_high_adu_extensions_to_notable():
         due_date = next(due_date_generator)
         try:
             # We can't use update_or_create because we need to pass _due_date to save
-            # TODO: promotedaddon; due date (Write PR)
-            promo = PromotedAddon.objects.get(addon_id=addon_id)
-            if promo.group_id != PROMOTED_GROUP_CHOICES.NOT_PROMOTED:
-                # Shouldn't happen because filter only includes NOT_PROMOTED.
+            promotions = PromotedAddonPromotion.objects.filter(
+                addon_id=addon_id
+            ).first()
```

Comment:
Should there be a todo to remove the first at some point, or why do we only do this for the first promotion? We are already iterating the value in the log no?